amplify/backend/function/fetchVoterRewards/src/utils.js

```diff
@@ -0,0 +1,92 @@
+const { providers, BigNumber } = require('ethers');
+const {
+  getColonyNetworkClient,
+  Network,
+  Extension,
+} = require('@colony/colony-js');
+const { default: fetch, Request } = require('node-fetch');
+
+/*
+ * @TODO These values need to be imported properly, and differentiate based on environment
+ */
+const API_KEY = 'da2-fakeApiId123456';
+const GRAPHQL_URI = 'http://localhost:20002/graphql';
+const RPC_URL = 'http://network-contracts.docker:8545'; // this needs to be extended to all supported networks
+
+const {
+  etherRouterAddress: networkAddress,
+} = require('../../../../mock-data/colonyNetworkArtifacts/etherrouter-address.json');
+
+const getNetworkClient = () => {
+  const provider = new providers.JsonRpcProvider(RPC_URL);
+
+  const networkClient = getColonyNetworkClient(Network.Custom, provider, {
+    networkAddress,
+    reputationOracleEndpoint:
+      'http://reputation-monitor.docker:3001/reputation/local',
+  });
+
+  return networkClient;
+};
+
+const getVotingClient = async (colonyAddress) => {
+  const networkClient = getNetworkClient();
+  const colonyClient = await networkClient.getColonyClient(colonyAddress);
+  return colonyClient.getExtensionClient(Extension.VotingReputation);
+};
```

Comment:
Is there no way to reuse these functions across the lambda functions? I think I've seen those in other functions and I'm curious if it's possible to reuse them

Reply:
@ArmandoGraterol There is something called Lamda layers which is intended to address this problem, but we aren't currently using it